.github/workflows/cibuildwheel-before-test.sh

```diff
@@ -8,12 +8,12 @@ PACKAGE_DIR=$1
 
 if [[ $(python --version 2>&1) == *"3.6."* ]]; then
   # Python 3.6 is only supported with oldest requirements
-  pip install -U -r "${PACKAGE_DIR}/.circleci/ci-oldest-reqs.txt" --progress-bar=off
+  pip install -U -r "${PACKAGE_DIR}/.github/workflows/conda-envs/requirements-test-oldest.txt" --progress-bar=off
 elif [[ $(python --version 2>&1) == *"3.7."* ]]; then
   # Python 3.7 was dropped by NEP 29 so not compatible with some of the newest dependencies
-  pip install -U -r "${PACKAGE_DIR}/requirements/requirements-test-compatible.txt" --progress-bar=off
+  pip install -U -r "${PACKAGE_DIR}/.github/workflows/conda-envs/requirements-test-compatible.txt" --progress-bar=off
```

Comment:
I would recommend dropping Python 3.7 rather than continuing to maintain a separate set of "compatible" dependencies.

Reply:
Good point. My plan was to re-evaluate which python versions we support once python 3.11 was released. Dropping support for python versions might be best for another PR, though.